examples/dashboards/query.json

```diff
@@ -1531,6 +1532,94 @@
                "renderer": "flot",
                "seriesOverrides": [ ],
                "spaceLength": 10,
+               "span": 12,
+               "stack": false,
+               "steppedLine": false,
+               "targets": [
+                  {
+                     "expr": "max_over_time(thanos_query_concurrent_gate_queries_max{job=~\"$job\"}[5m]) - avg_over_time(thanos_query_concurrent_gate_queries_in_flight{job=~\"$job\"}[5m])",
```

Comment:
```suggestion
                     "expr": "max_over_time(thanos_query_concurrent_gate_queries_max{job=~\"$job\"}[$__rate_interval]) - avg_over_time(thanos_query_concurrent_gate_queries_in_flight{job=~\"$job\"}[$__rate_interval])",
```

Reply:
Thank you, Filip :) The PR has been updated accordingly.
In the same file there is a variable `$interval`. Is it equivalent to `$__rate_interval` ?